### DISCUSSION TITLE
More space for quick access to tags.

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#185] [Feature Requests]
[VP]: I often have to work with tags, but it takes a long time to get to them. I want to do it in one step.
![5b371099cc2910363f9487803461d60c](https://github.com/user-attachments/assets/35543b99-03ea-4dca-bad6-25d4675f5b4b)

**Comment by irmod** [comment]
I suggest creating four tabs on the left Navigation Drawer.

One of the tabs can display **All Tags**. This tab will allow you to add tag management (rename, set the color for a tag, etc.). Sorting must be available for tags. Tags must be searchable. Below I have added a video with an example.  

Select a separate tab to manage **Saved Searches**. 

Separate tab for navigation and management:
 - Calendar
 - Gallery
 - Repositories
 - Attributes
 - Plugins
 - Recycle Bin

Make this tab the Default.  Allow the user to set their Default tab selection in Settings.

On the fourth tab (let's call it "**Shelf**"), place a list of open documents (notes) for quick navigation through them.  This may be similar to vertical tabs in a browser.

Follow this link for a video of how tag search works and a general example of tabs (let's imagine that a tab displays **All Tags**)

https://github.com/user-attachments/assets/f71fec1d-ef38-446e-8cac-50b8d0ac8b01
